### ISSUE TITLE
twisted.test.test_assertions is lonely

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2689]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2689|
|Type|defect|
|Created|2007-06-03 21:36:15Z|
|Branch|https://github.com/twisted/twisted/tree/lonely-assertions-2689|

This was added before the recent push to improve trial.  Presumably it was passed over when trial's test suite was moved into twisted.trial.test.  Hopefully everything it covers is covered elsewhere now, in which case it can just be deleted.  Otherwise, it should be moved into the trial test suite.

<details><summary>Searchable metadata</summary>

```
trac-id__2689 2689
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_lonely_assertions_2689 branches/lonely-assertions-2689
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1180906575000000 1180906575000000
changetime__1262835118000000 1262835118000000
version__None None
owner__ 
cc__jesstess cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_1"></a>|@jesstess commented|
|-|-|

(In [27889]) Branching to 'lonely-assertions-2689'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [27893]) Remove twisted/test/test_assertions.py as the functionality is
duplicated in twisted/trial/test/test_assertions.py.

refs [#2689](https://github.com/twisted/twisted/issues/2689)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess removed owner|
|-|-|

Changes in the branch:
 * Delete test_assertions.py. It had non-specifics tests for assertRaises and the errors ZeroDivisionError and ValueError, all of which are covered in the trial test_assertions.py
 * Make docstrings in the trial test_assertions.py conform to standard
 * Update copyright on the trial test_assertions.py

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @jesstess|
|-|-|

Copyright should be 2010 ;)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess removed owner|
|-|-|

Copyright updated.

**Comment by twisted-trac** [comment]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|TimAllen set owner to TimAllen|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_8"></a>|TimAllen set owner to @jesstess|
|-|-|

Somewhat oddly, the deleted test tested `assertRaises` but the new tests don't — just lots of tests for `failUnlessRaises`. On the other hand, `assertRaises` is just an alias for `failUnlessRaises`, and the new tests actually _use_ `assertRaises` quite a lot, so I guess it's actually getting tested anyway.

I say this branch is ready to merge!

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_9"></a>|@jesstess set status to `closed`|
|-|-|

(In [27916]) Merge lonely-assertions-2689

Author: jesstess
Reviewer: TimAllen
Fixes: [#2689](https://github.com/twisted/twisted/issues/2689)

twisted/test/test_assertions.py can be deleted as the functionality it
exercises is covered in trial/test/test_assertions.py.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
